zhangyan python homework

zhangyan0814 opened the pull request.

zhangyan0814 committed:
Update armstrong_numbers.py

zhangyan0814 committed:
Update rotational_cipher.py

zhangyan0814 committed:
Update difference_of_squares.py

zhangyan0814 committed:
Update anagram.py

zhangyan0814 committed:
Update allergies.py

cr4ftsm4n commented on python/armstrong-numbers/armstrong_numbers.py:
```diff
@@ -1,2 +1,10 @@
+    if sum == number:
+        return True
+    else:
+        return False
```
equals
```python
return  sum == num
```

cr4ftsm4n reviewed (commented):
LGTM

cr4ftsm4n reviewed (commented):
97はord("A")をコメント欄に書かないと、あとで読むとわからいよね

zhangyan0814 committed:
Update series.py

zhangyan0814 committed:
Update robot_simulator.py

zhangyan0814 committed:
Update sieve.py

zhangyan0814 committed:
Update atbash_cipher.py

zhangyan0814 committed:
Update sum_of_multiples.py

zhangyan0814 commented:
zhangyan homework

zhangyan0814 committed:
Update acronym.py

zhangyan0814 committed:
Update say.py

zhangyan0814 committed:
Update largest_series_product.py

zhangyan0814 committed:
Update kindergarten_garden.py

zhangyan0814 committed:
Update grade_school.py

zhangyan0814 commented:
python homework

zhangyan0814 committed:
Update luhn.py

zhangyan0814 committed:
Update grains.py

zhangyan0814 committed:
Update space_age.py

zhangyan0814 committed:
Update roman_numerals.py

zhangyan0814 committed:
Update roman_numerals_test.py

test source 
changed numeral to numerala because method name is also numeral
if I don't change the name it will be error

zhangyan0814 committed:
Update flatten_array.py

zhangyan0814 committed:
Update etl.py

zhangyan0814 committed:
Update prime_factors.py

zhangyan0814 committed:
Update pig_latin.py

zhangyan0814 committed:
Update simple_cipher.py

zhangyan0814 committed:
Update scrabble_score.py

zhangyan0814 committed:
Update crypto_square.py

zhangyan0814 committed:
Update sublist.py

zhangyan0814 committed:
Update pythagorean_triplet.py

zhangyan0814 committed:
Update circular_buffer.py

zhangyan0814 committed:
Update robot_name.py

zhangyan0814 committed:
Update rail_fence_cipher.py

zhangyan0814 committed:
Update matrix.py

zhangyan0814 committed:
Update beer_song.py

zhangyan0814 committed:
Update saddle_points.py

zhangyan0814 committed:
Update nth_prime.py

zhangyan0814 committed:
Update bracket_push.py

zhangyan0814 committed:
Update twelve_days.py

zhangyan0814 committed:
Update secret_handshake.py

zhangyan0814 committed:
Update perfect_numbers.py

zhangyan0814 committed:
Update palindrome_products.py
